diwako_cbrn.vr/scripts/cbrn/functions/fn_detectorGeigerPFH.sqf

```diff
@@ -0,0 +1,35 @@
+if (!isGameFocused || isGamePaused) exitWith {};
+params ["_args", "_idPFH"];
+_args params ["_lastIteration"];
+
+private _time = cba_missiontime;
+private _delta = _time - _lastIteration;
+_args set [0, _time];
+
+if ( !cbrn_geiger ||  {!(ace_player getVariable ["cbrn_detector_geiger", false])} ) exitWith {
+    cbrn_geigerPfh = -1;
+    [_idPFH] call CBA_fnc_removePerFrameHandler;
+};
+
+if (cbrn_curThreat < 0.25) exitWith {};
+
+private _wait = 2;
+
+if ((cbrn_lastGeiger + _wait) <= _time) then {
+    cbrn_lastGeiger = _time;
```

Comment:
Suggestions, do write `cbrn_lastGeiger` as when you area allowed to do another sound. This means you do not need to add 2 to your value and them compare. This is just a very small improvement mind you.